WHBNDL/Infrastructure/AppTextsDeserializer.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Text.Json;
+
+using WHBNDL.Domain;
+
+namespace WHBNDL.Infrastructure
+{
+    internal class AppTextsDeserializer
+    {
+        public static Texts Deserialize()
+        {
+            string path = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"..\..\..\", "Resources", "appTexts.json");
```

Comment:
Huhh, ez a sok ..\ picit fura megoldás és akár veszélyes is tud lenni. Pl ha a programodat közvetlenül a C:\ meghajtóra másolom. Inkább a Resources mappát másolnám a BaseDirectory mellé deployment esetben.